run_kraken.py

```diff
@@ -37,9 +39,28 @@ def main(cfg):
     logging.info("Starting kraken")
 
     # Parse and read the config
-    if os.path.isfile(cfg):
-        with open(cfg, "r") as f:
-            config = yaml.full_load(f)
+    # Initializing flag variable as 0 to test config is a file or not
+    flag = 0
+
+    try:
+        # For config as remote file
+        if urlparse(cfg):
+            # Set flag as 1 since config is remote file
+            flag = 1
+            content = requests.get(cfg)
+            texts = content.text
+            config = yaml.safe_load(texts)
+
+    except:
```

Comment:
As indicated by pre-commit check, bare 'except' should not be used.

Reply:
also good to print off the exception you got, like this example [here](https://github.com/cloud-bulldozer/kraken/blob/adb465cab02f7b1ecc289e99f3c8d4ed773ca08a/kraken/node_actions/aws_node_scenarios.py#L26)